README.md

```diff
@@ -111,34 +98,21 @@ mount / render it into the HTML element you created in step 3.
     }    
     ```
 
-5. Run `pub serve` in the root of your Dart project.
+5. Run `pub run build_runner serve` in the root of your Dart project.
 
 &nbsp;
 
 ### Running tests in your project
 
-When running tests on code that uses our [transformer] _(or any code that imports `over_react`)_,
-__you must run your tests using Pub__.
-
-1. Add the `test/pub_serve` transformer to your `pubspec.yaml` _after_ the `over_react` transformer.
-
-    ```yaml
-    transformers:
-      - over_react
-      - test/pub_serve:
-          $include: test/**_test{.*,}.dart
-      - $dart2js
-    ```
+When running tests on code that uses our [builder] _(or any code that imports `over_react`)_,
+__you must run your tests using build_runner__.
```

Comment:
Should add a huge, bold callout here that any running `build_runner watch` or `build_runner serve` tasks should be killed before running `build_runner test` to avoid destruction of the cached asset graph.

https://github.com/dart-lang/build/issues/352#issuecomment-461554316